python: can't open file '/tmp/gptme-evals-lyclilra/hello.py': [Errno 2] No such file or directory
